strix/utilities/utils.py

```diff
@@ -81,6 +81,33 @@ def parse_datalist(filelist, format="auto", has_unlabel: bool = False):
         unlabel: str = DatalistKeywords.UNLABEL.value
         if label not in datalist:
             raise GenericException(f"Your datalist does not contain '{label}' key.")
+        else:
```

Comment:
You should add a `elif` condition instead of a `else`, this will cause the program skip original processing below (Line 111).